src/SDKs/Search/DataPlane/Microsoft.Azure.Search.Data/Customizations/Documents/DocumentsOperationsExtensions.cs

```diff
@@ -27,9 +27,10 @@ public static class DocumentsOperationsExtensions
         /// </param>
         public static long Count(
             this IDocumentsOperations operations,
-            SearchRequestOptions searchRequestOptions = default(SearchRequestOptions))
+            SearchRequestOptions searchRequestOptions = default(SearchRequestOptions),
+            Dictionary<string, List<string>> customHeaders = null)
```

Comment:
Please make these consistent with the conventions used by the generated methods that we expose in the rest of the SDK. These only take `customHeaders` in the core `WithHttpMessagesAsync` methods. Take a look [here](https://github.com/Azure/azure-sdk-for-net/blob/psSdkJson6/src/SDKs/Search/DataPlane/Microsoft.Azure.Search.Service/Generated/DataSourcesOperationsExtensions.cs) for examples.